crypto/src/mls/conversation/conversation_guard/commit.rs

```diff
@@ -42,8 +37,15 @@ impl ConversationGuard {
         }
     }
 
+    pub(super) async fn merge_commit(&mut self) -> Result<()> {
+        let client = self.session().await?;
+        let backend = self.crypto_provider().await?;
```

Comment:
[Feel free to ignore this comment, I'm just mentioning it here for the future PRs and as a general comment on existing code.]

I'd prefer to avoid the word `backend` for the crypto provider, as it can be at first confusing (there is Wire backend). Possible alternatives: `provider`, `crypto_provider`...